src/index.js

```diff
@@ -62,7 +64,7 @@ class SpriteAnimator extends Component {
         onLoad()
         this.setState({
           isLoaded: true,
-          maxFrames: Math.floor(direction === 'horizontal' ?
+          maxFrames: frameCount || Math.floor(direction === 'horizontal' ?
```

Comment:
`frameCount` needs to be specified for multi-dimensional sprites.

**example**: let's assume we have a spritesheet with 11 frames and 4 columns (`frameCount: 11, wrapAfter: 4`). this would result in an image with 2 rows x 4 columns + 1 row x 3 columns. there is no way to correctly infer the frame count just from the spritesheet image dimensions, hence the new `frameCount` prop.